### ISSUE TITLE
V4.1.4 The top bar button is gone

### ISSUE THREAD

**Issue by jarodise** [hkgnp/logseq-todoist-plugin#26]
Just updated to the V4.1.4 version, now the button on the top-bar is nowhere to be found.

**Comment by hkgnp** [comment]
Hi all,
The configuration can now be found within the plugin settings section, instead of the top bar!

**Comment by MattHulse** [comment]
Does that require a new update since yesterday?  That's the first place I looked and it wasn't there.

**Comment by hkgnp** [comment]
Hmm, the updated one that does not have the button in the top bar should be the one. Are you on the latest Logseq version as well?

**Comment by MattHulse** [comment]
Yes, I upgraded to 0.6.9 before I installed the plugin.  I also just uninstalled and reinstalled todoist plugin.  It is at version 4.1.6.  Under Settings -> Plugin Settings I see entries for 2 other plugins I currently have installed, but not todoist.

Update: I just tried to close LogSeq completely and reopen and still no settings for todoist plugin.

**Comment by hkgnp** [comment]
Could you:
1) Uninstall the plugin
2) Close Logseq
3) Open Logseq
4) Install plugin
5) Restart Logseq

**Comment by MattHulse** [comment]
That didn't change anything for me.

I did turn on developer tools and I see the following in the console when I force a reload of LogSeq:
![image](https://user-images.githubusercontent.com/56975/169434558-6e234c30-c53e-4c8a-a1b2-6aab2eb3ac6d.png)

It seems like it's trying to talk to todoist on startup, before I have been able to configure the plugin with my todoist key.  I wonder if this keeps it from loading the plugin settings?


**Update**: So I went and set the apiKey manually in the settings file and reloaded logseq and everything loaded correctly.  I can now see the plugin settings tab for todoist.  I think it's just an issue with loading the very first time, before there is a key in the settings.
